userguide/aws/how-tos/add-a-certificate.adoc

```diff
@@ -0,0 +1,157 @@
+= How To Add a Certificate
+
+== Overview
```

Comment:
I think we can get rid of the explanatory information about what HTTPS is. For someone like me, it was informative, but for the reader we are targeting, they should already be aware of this information. The only information I would keep would be the (edited) second paragraph.

Reply:
Also, see the Quick Start with Kaui guide (new) with the high-level steps in the beginning.

Reply:
OK but maybe the brief summary of HTTPS should go in a (Divio style) explanation document? It seems that those may be useful for the reader who wants more information, while others can skip them.

Reply:
Since you have already written the material, you could break it out into an explanation document and link to the explanation document from this how-to document.

Reply:
Ok that has been done.